iiif-5-day-workshop/day-one/iiif-introduction.md

```diff
@@ -1 +1,133 @@
-# IIIF Introduction
+# Introduction to IIIF
+
+
+## The Problem
+
+![image](https://raw.githubusercontent.com/IIIF/training/master/iiif-5-day-workshop/images/data%20silos.jpg)
```

Comment:
Can you change these urls to relative urls?

```md
![image](../images/data%20silos.jpg)
```
or something similar?

Reply:
Can do !